app/controllers/forms_controller.rb

```diff
@@ -43,6 +43,18 @@ def update
     end
   end
 
+  def new
+    form = Form.create! name: 'new-form'
```

Comment:
Where working on changing the method now so we can make the new form without having a name pre-sett name.